examples/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nginx:1.18.0-alpine
```

Comment:
What is this Dockerfile for? Please add some comment.

Do we have to put it in the `examples` directory?

Reply:
Added comment. We can put it in the main directory if that isn't an issue.